docs/component.md

```diff
@@ -12,18 +12,6 @@ let masked = false
 <display :value="value" />
 ```
 
-```js
-let franceIBAN = 'FR7630006000011234567890189'
-let masked = true
-
-<example label="France IBAN">
-  <input-facade mask="FR## #### #### #### #### #### ###" v-model="franceIBAN" :masked="masked" />
-</example>
-
-<checkbox v-model="masked" />
-<display :value="franceIBAN" />
-```
-
```

Comment:
Why should we remove this example?

Reply:
This change has already been merged to master from [this commit](https://github.com/RonaldJerez/vue-input-facade/commit/b3655e21f7545c6adadb255a04f02d2bc9fd75c6#diff-09a78bead7fb056ad287a451d6c20c5d349b512c453ac18d93375f5bb201cf5fL16). The beta branch is behind master so these commits are also showing here. I am open to adding it back if desired but seems like it is another basic usage example where we still kept the US phone number example.